### PR TITLE
Fixes #34466 - Fix playbooks endpoint to return a string

### PR DESCRIPTION
**Pull request by adiabramovitch**
Related to theforeman/foreman_ansible#506

### CODE REVIEW

**Comment by adiabramovitch** [comment]
@ezr-ondrej The fix :)

**Review by ezr-ondrej — COMMENTED** [review]
I believe we have some tests testing the output of this method, so we will need to change those as well :)

File: test/playbooks_reader_test.rb [hunk 1]
```diff
@@ -4,90 +4,64 @@
 require_relative '../lib/smart_proxy_ansible/reader_helper'
 
 class PlaybooksReaderTest < Minitest::Test
+  INPUT = JSON.parse(File.read(File.join(__dir__, 'fixtures/playbooks_reader_data.json')))
```
Comment (ezr-ondrej) [inline test/playbooks_reader_test.rb:7]: could we have this as 
```suggestion
  let(:fixtures) { JSON.parse(File.read(File.join(__dir__, 'fixtures/playbooks_reader_data.json'))) }
```

The global variable seems like not the best practice for a dynamic value based on a file.

File: test/playbooks_reader_test.rb [hunk 2]
```diff
@@ -5,89 +5,63 @@
...
-                       "  vars:\n",
-                       "    libvirt_tftp: true\n",
-                       "  roles:\n",
-                       "    - foreman\n",
-                       "    - libvirt\n",
-                       "    - foreman_provisioning\n"]
+  describe 'playbooks method' do
+    let(:fixtures) { JSON.parse(File.read(File.join(__dir__, 'fixtures/playbooks_reader_data.json'))) }
+    describe 'playbooks method with error' do
+      let(:ansible_config) { fixtures['ansible_config'] }
```
[the remark this answers is not in the document]
Comment (adiabramovitch) [reply test/playbooks_reader_test.rb:11]:
```
describe 'playbooks method' do
    let(:fixtures) { JSON.parse(File.read(File.join(__dir__, 'fixtures/playbooks_reader_data.json'))) }
    let(:ansible_config) { fixtures['ansible_config'] }'
    describe 'playbooks method with error' do 
```
  @ezr-ondrej  do you mean like that?

File: test/playbooks_reader_test.rb [hunk 3]
```diff
@@ -5,89 +5,63 @@
...
+    end
   end
 
-  test 'raises error if the playbooks path is not readable' do
-    Proxy::Ansible::PlaybooksReader.expects(:read_collection_playbooks)
-                                   .with('/etc/ansible/collections', nil).raises(Proxy::Ansible::ReadPlaybooksException.new('Could not read Ansible roles'))
-    ex = assert_raises(Proxy::Ansible::ReadPlaybooksException) do
-      Proxy::Ansible::PlaybooksReader.playbooks(nil)
+  describe 'playbooks_names method' do
+    let(:ansible_config) { fixtures['ansible_config'] }
```
Comment (ezr-ondrej) [inline test/playbooks_reader_test.rb:57]: Should we remove these now? it is the same as the global one, so no need to override it now, right?

**Review by ezr-ondrej — APPROVED** [review]
Thanks @adiabramovitch ! :+1:

**Pull request merged**